code/__DEFINES/turfs.dm

```diff
@@ -4,3 +4,9 @@
 #define TURF_PATHING_PASS_PROC 1
 /// Turf is never passable
 #define TURF_PATHING_PASS_NO 2
+
+/// Returns a list of turfs similar to CORNER_BLOCK but with offsets
+#define CORNER_BLOCK_OFFSET(corner, width, height, offset_x, offset_y) ((block(locate(corner.x + offset_x, corner.y + offset_y, corner.z), locate(min(corner.x + (width - 1) + offset_x, world.maxx), min(corner.y + (height - 1) + offset_y, world.maxy), corner.z))))
```

Comment:
block now accepts direct coords, and its faster
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/9c07a501-91ae-4590-9dde-88b93b13363a)
https://www.byond.com/docs/ref/#/proc/block

Reply:
Fixed, PTAL